packages/client/manifest.json

```diff
@@ -2350,6 +2350,12 @@
             "value": "above"
           }
         ]
+      },
+      {
+        "type": "section",
+        "label": "Type",
+        "key": "type",
+        "defaultValue": "oneColumn"
```

Comment:
We could make this new setting depend on a label position value of "above". Then users will only see it when they're actually able to use it.